socketio/src/asynchronous/client/client.rs

```diff
@@ -854,7 +847,7 @@ mod test {
             .await?;
 
         // open packet
-        let mut socket_stream = socket.as_stream();
+        let mut socket_stream = socket.as_stream().await;
         let _ = socket_stream.next().await.unwrap()?;
 
         println!("Here12");
```

Comment:
Just saw this now: Where does that come from? :)

Reply:
It wasn't me. Apparently you added it 12 months ago: https://github.com/1c3t3a/rust-socketio/pull/296/files#diff-dd5d6ff18628a21723847133d6e54ab531454037b62be81d2c42109d333aeab9R609 😆 

I'll remove it.

Reply:
Done.